tutorials-and-examples/inference-servers/jetstream/maxtext/maxengine-server/Dockerfile

```diff
@@ -4,6 +4,8 @@
 FROM ubuntu:22.04
 
 ENV DEBIAN_FRONTEND=noninteractive
+ENV MAXTEXT_VERSION=jetstream-v0.2.0
```

Comment:
How is the compatibility of these releases managed today, across the MaxText and JetStream repos?

Reply:
There is no cadence for the Jetstream release, and MaxText repository does not do releases, they will only tag their repository.